src/Tabs.js

```diff
@@ -20,6 +20,7 @@ const propTypes = {
    * @controllable onSelect
    */
   activeKey: PropTypes.any,
+  defaultActiveKey: PropTypes.any,
```

Comment:
Needs prop type comments for documentation purposes.

Reply:
is this actually a prop, even?

Reply:
it's returned from `uncontrollable`, no?

Reply:
ah, consumed by uncontrollable, even. i got a little confused given the `getDefaultActiveKey` function, which is doing something somewhat unrelated

Reply:
can copy the comment from accordion

Reply:
There is no docs provided, see https://5c507d49471426000887a6a7--react-bootstrap.netlify.com/components/tabs/, although the examples use it, and my experience shows it is necessary...

Reply:
> There is no docs provided
okay, so add the comments for it then.

Reply:
done